pkg/console/clientwrapper/without_secret.go

```diff
@@ -0,0 +1,39 @@
+package clientwrapper
+
+import (
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+func WithoutSecret(client kubernetes.Interface) kubernetes.Interface {
+	return &clientWrapper{
+		Interface: client,
+		core: &coreWrapper{
+			CoreV1Interface: client.CoreV1(),
+			secrets:         fake.NewSimpleClientset().CoreV1(),
```

Comment:
This may need another wrapper to make sure it correctly fools the informer but I will let you guys figure it out.